Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -518,7 +519,7 @@ export default function AssetPage(props: IProps) {
                                     onChange={(e) => setSearchTerm((prev) => ({ ...prev, channels: e.target.value }))}
                                     style={{ width: '100%', marginBottom: '10px' }}
                                 />
-                                <select multiple style={{ height: innerHeight - 330, width: '100%' }} onChange={(evt) => {
+                                <select multiple style={{ height: '80%', width: '100%' }} onChange={(evt) => {
```

Comment:
use `flex` for these. Otherwhsie they depend on the height of the `Input` which may be changing